helm/stargate/templates/stargate_docsapi_deployment.yaml

```diff
@@ -0,0 +1,51 @@
+{{ if .Values.docsapi.enabled }}
```

Comment:
same stuff as in the coordinator:

* probes should be faster, why 80s
* limits and requests could be the same
* affinity